AnnoyancesFilter/sections/css_extended.txt

```diff
@@ -96,6 +96,7 @@ facebook.com,facebookcorewwwi.onion#%#(function() { var observer=new MutationObs
 eva.ru##div:has(> div > .subscribe-form)
 finanzen.net##.teaser > .box:has(> .box-headline:contains(WhatsApp Newsletter))
 finanzen.net##.col-sm-4 > .box:has(> .box-headline:contains(WhatsApp Newsletter))
+xakep.ru##.widget_custom_html:has(> .block-title > span:contains(Еженедельный дайджест))
```

Comment:
I would suggest to check for ID mc_embed_signup instead of text as text may change easier in the future. Or even better the form action.
form[action*="list-manage.com/subscribe"]